src/main/java/homework_2/traffic_light/TrafficLight.java

```diff
@@ -0,0 +1,44 @@
+package homework_2.traffic_light;
+
+import java.time.Instant;
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class TrafficLight {
+
+    public static final String ERROR_MESSAGE = "Only 1 non-negative integer is allowed as passed parameter";
+    public static final String DAY_OVER = "The day is over";
+
+    public void run() {
```

Comment:
Here and in other places: i would recommend getting rid of multiple if/else statements and separate logic to other methods